config/google_cloud_exporter/docker/README.md

```diff
@@ -10,10 +10,60 @@ See the [prerequisites](../prerequisites.md) doc for Google Cloud prerequisites.
 
 **Docker Socket**
 
-The provided configuration assumes the collector is running on the Docker system. By default, the `endpoint` collector from is `unix:///var/run/docker.sock`.
+The provided configuration assumes the collector is running on the Docker system. By default, the `endpoint` collected from is `unix:///var/run/docker.sock`.
 
 The user running the collector must have permission to read the [docker socket](https://docs.docker.com/engine/install/linux-postinstall/).
 
 ```bash
 sudo usermod -aG docker observiq-otel-collector
 ```
+
+## Deployment
+
+**On Host**
+
+If the agent is running on the Docker host (not within a container), deployment is as simple as installing the agent
+and updating the configuration.
```

Comment:
should this be on a separate line?

Reply:
I do not understand. Link to the installation documentation perhaps?